packages/plugin-ext/src/common/plugin-protocol.ts

```diff
@@ -157,6 +157,31 @@ export interface PluginPackageLanguageContributionConfiguration {
 }
 
 export const PluginScanner = Symbol('PluginScanner');
+/**
```

Comment:
I've moved an interface to the symbol to easy navigation, no changes to the interface were made.